Backpack/Button/Classes/BPKButton.m

```diff
@@ -374,7 +374,8 @@ - (void)updateContentColor {
         [self setAttributedTitle:nil forState:UIControlStateNormal];
     }
     
-    
+
+    self.tintColor = self.currentContentColor;
```

Comment:
This seems to be required to make the tint colour be correct when first rendered 🤔